### DISCUSSION TITLE
Heltec WiFi_LoRa32_V3

### DISCUSSION THREAD

**Discussion by aramlakhan** [cyberman54/ESP32-Paxcounter#946] [Q&A] [answered]
Hi,
Has anyone got this board up and running? What settings did you change?

**Comment by cyberman54** [comment, marked as answer]
[Heltec Website](https://heltec.org/project/wifi-lora-32-v3/) says this board has a SX1262 Lora chip. Unfortunately, the LORAWAN stack, which the Paxcounter code relies upon, MCCI LMIC, [so far does not](https://github.com/mcci-catena/arduino-lmic/issues/301) support this chip. Thus, there is currently no way to get this up and running.

**Reply by aramlakhan** [reply]
thanks for the info
